### PR TITLE
Add cloud.vars to readme

### PR DESCRIPTION
**Pull request by ghost**
Fix for #7

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "scratchcloud",
-  "version": "1.1.0",
+  "version": "1.1.1",
```
Comment (AmazingMech2418) [inline package.json]: No need to bump version if just a documentation edit.

File: README.md [hunk 2]
```diff
@@ -72,4 +72,6 @@ const {Session, Cloud} = require('scratchcloud');
 ## Other Information
 You can use the `cloud.get(variableName)` to get the value of a cloud variable by name.
 
-Also, note that you can use the cloud symbol in your variable names, but you do not have to.
+Use `cloud.vars` to get all the names and values of all the cloud variables in a project.
```
Comment (AmazingMech2418) [inline README.md]: Maybe mention that it returns an object with the variable names with the cloud symbol as keys, and the values of these variables as the values?

**Review by AmazingMech2418 — CHANGES_REQUESTED** [review]
Just a few changes I would like to request...

**Comment by ghost** [comment]
Resolved.

**Pull request merged**